lib/Dialect/TritonGPU/IR/LinearLayoutConversions.cpp

```diff
@@ -926,6 +937,7 @@ std::optional<LinearLayout> chooseStMatrixLayoutNoLeadingOffset(
   StringAttr kWarp = S("warp");
   StringAttr kCol = S("dim1");
   StringAttr kRow = S("dim0");
+  StringAttr kBlock = S("block");
```

Comment:
bikeshedding: I hope the name is explicit enough. Block term is way overloaded unfortunately. We could name it CTA but that's very nvidia specific.

Reply:
Sure, I had a try to change it to CTA but found it a bit annoying to change a lot of places. Let me sort it out in the future.